plugins/spotify/start.sh

```diff
@@ -25,12 +25,19 @@ if [[ -n "$SOUND_SPOTIFY_USERNAME" ]] && [[ -n "$SOUND_SPOTIFY_PASSWORD" ]]; the
     --password "$SOUND_SPOTIFY_PASSWORD"
 fi
 
+# SOUND_SPOTIFY_DISABLE_CACHE: Disable Spotify audio cache
+if [[ ! -z "$SOUND_SPOTIFY_DISABLE_CACHE" ]]; then
```

Comment:
Small nitpick here. If I recall correctly:
- `-z`: tests if length is zero
- `-n`: tests if length is not zero

So `! -z` can be replaced with `-n` which is a bit cleaner imo.

Reply:
I agree with ```-n``` being cleaner, but IIRC I couldn't get it to work as expected.  That said, I was learning bash, Github, and balenaSound architecture all at the same time, so it might have been something on my end.